lib/rubocop/config_validator.rb

```diff
@@ -270,21 +270,31 @@ def check_cop_config_value(hash, parent = nil)
           supposed_values = 'a boolean'
         elsif key == 'AutoCorrect' && !CONFIG_CHECK_AUTOCORRECTS.include?(value)
           supposed_values = '`always`, `contextual`, `disabled`, or a boolean'
+        elsif key == 'Reference'
+          warn string_reference_warning(parent)
```

Comment:
Regardless of if the option is renamed or not, I would prefer this to not warn immediatly. Other RuboCop extensions, including first-party ones, currently specify it as a string and they would need to rush out an update in order to not warn.
https://github.com/search?q=org%3Arubocop%20%2FReference%3A%20%2F&type=code